src/test/java/POM/pages/BasePage.java

```diff
@@ -43,4 +45,8 @@ public BasePage clickFavorites() {
         wait.until(ExpectedConditions.elementToBeClickable(favoritesMenuItem)).click();
         return this;
     }
+    public BasePage clickArtists() {
+        wait.until(ExpectedConditions.elementToBeClickable(artistsMenuItem)).click();
+        return this;
+    }
```

Comment:
**Base Page \ Base Class** — Implements the necessary methods for work with webdriver.

We will store the web elements present on each page in the corresponding java class.

The methods:  

1)	`clickAllSongs()`  must be stored in the `class AllSongsPage`
2)	`clickAlbums()` must be stored in the `class AlbumsPage`
3)	`clickFavorites()` must be stored in the `class FavoritesPage`
4)	`clickArtists()` must be stored in the `class ArtistsPage`

![image](https://user-images.githubusercontent.com/104761482/205430256-426e9e1f-e309-4513-b960-2636358180d4.png)